### PR TITLE
Add API for developers to update Alias

### PR DESCRIPTION
**Pull request by poliarysh**

### CODE REVIEW

File: ezpush-chrome.js [hunk 1]
```diff
@@ -231,4 +240,35 @@ var EzPush = {};
         EzPush.log('Send: '+ window.JSON.stringify(json));
         EzPush.socket.send(window.JSON.stringify(json));
     }
+
+    /***** Api methods ******/
+    function updateUserId(param) {
```
Comment (ronenhamias) [inline ezpush-chrome.js]: change the function signeture to
setUserAlias(alias)

File: ezpush-chrome.js [hunk 2]
```diff
@@ -231,4 +240,35 @@ var EzPush = {};
...
+
+        var updateUserId = {
+            "qualifier": "pt.openapi.push.devreg/updateUserId",
+            "contextId": EzPush.contextId,
+            "data": {
+                "deviceRegistrationId": {
+                    "hwid": EzPush.subscriptionId,
+                    "applicationId": param.id
+                },
+                "userIdentity": param.identity
```
Comment (ronenhamias) [inline ezpush-chrome.js]: userIdentity": alias

File: ezpush-chrome.js (hunk 1 above)
Comment (poliarysh) [inline ezpush-chrome.js]: Fixed;

Comment (poliarysh) [inline ezpush-chrome.js]: Removed applicationId param, we already have it as ezPush.appId;

File: ezpush-chrome.js (hunk 2 above)
Comment (poliarysh) [inline ezpush-chrome.js]: Fixed;

**Pull request merged**